academy/exchange/cloud/authenticate.py

```diff
@@ -69,30 +73,51 @@ class GlobusAuthenticator:
         audience: Intended audience of the token. This should typically be
             the resource server of the the token was issued for. E.g.,
             the UUID of the ProxyStore Relay Server application.
-        auth_client: Optional confidential application authentication client
-            which is used for introspecting client tokens.
     """
 
+    TOKEN_CACHE_LIMIT: int = 1024
+    TOKEN_TTL: int = 60
```

Comment:
Can you append the unit to this var? I.e., `TOKEN_TTL_SECONDS`

Reply:
I guess there is some tradeoff here between performance and guaranteeing the validity of the token. 60 seconds seems ok to me.